js/ui/widget/ui.search_box_mixin.js

```diff
@@ -146,10 +150,19 @@ module.exports = {
         return searchMode === "equals" ? "=" : searchMode;
     },
 
+    _cleanAria: function($target) {
+        this.setAria({
+            "role": undefined,
+            "activedescendant": undefined
+        }, $target);
+        domUtils.toggleAttr($target, "tabIndex");
```

Comment:
Pull 19.1 branch. I removed this method some time ago.
Use `$.attr()` instead of it